app/controllers/routes_controller.rb

```diff
@@ -39,10 +40,17 @@ def destroy
   private
 
   def route_params
-    params.require(:route).permit(:name, railway_station_ids: [])
+    params.require(:route).permit(railway_station_ids: [])
   end
 
   def set_route
     @route = Route.find(params[:id])
   end
+
+  def station_position(stations)
+    RailwayStationsRoute.where(route: @route).delete_all
```

Comment:
Is it a good solution to delete  _RailwayStationsRoute_ and than create it again?

Reply:
Active Record can not update position column without primary key in this table. So I delete previous records.